### DISCUSSION TITLE
Services in CFI Repo

### DISCUSSION THREAD

**Discussion by AdrianHammond** [finos/compliant-financial-infrastructure#304] [General] [answered]
There are a number of services in main CFI repo e.g. Lamda, Redshift that have not been touched for 10 month. Should we have discussion between the Policy and RI WG to agree an approach to review and clean out services that are no longer being maintained. @abdullahgarcia @thinkl33t @eddie-knight

**Comment by eddie-knight** [comment]
Yep. I would suggest doing a cleanup & then create issues capturing the last commit with that content for reference. 

Eventually everything needs to be moved out, and we can just review the commit history when someone gets to building the associated modules.

**Comment by AdrianHammond** [comment, marked as answer]
Discussed in RIWG meeting on 25th Feb, @ml4 will pick this up. Will requests new rep and move content across.
